WalletWasabi.Fluent/Views/MainView.axaml

```diff
@@ -3,56 +3,60 @@
-            <rxui:RoutedViewHost Router="{Binding Router}">
-              <rxui:RoutedViewHost.ViewLocator>
-                <vl:ScreenViewLocator />
-              </rxui:RoutedViewHost.ViewLocator>
-              <rxui:RoutedViewHost.PageTransition>
-                <CrossFade Duration="0:0:0.25" />
-              </rxui:RoutedViewHost.PageTransition>
-              <rxui:RoutedViewHost.DefaultContent>
-                <TextBlock Text="Default content"
-                           HorizontalAlignment="Center"
-                           VerticalAlignment="Center" />
-              </rxui:RoutedViewHost.DefaultContent>
-            </rxui:RoutedViewHost>
-          </DockPanel>
+          <ExperimentalAcrylicBorder DockPanel.Dock="Left" IsHitTestVisible="False">
+            <ExperimentalAcrylicBorder.Material>
+              <ExperimentalAcrylicMaterial BackgroundSource="Digger"
+                                          TintColor="{DynamicResource SystemAltHighColor}"
+                                          TintOpacity="1"
+                                          FallbackColor="{DynamicResource AcrylicFallbackColor}"
+                                          MaterialOpacity="0.55" />
+            </ExperimentalAcrylicBorder.Material>
+
+          </ExperimentalAcrylicBorder>
+          <local:NavBar DataContext="{Binding NavBar}" />
         </Panel>
-      </DockPanel>
-    </Panel>
-  </SplitView>
+      </SplitView.Pane>
+      <Panel>
+        <DockPanel>
+          <local:StatusBar DockPanel.Dock="Bottom" DataContext="{Binding StatusBar}" VerticalAlignment="Bottom" Height="32" TextBlock.FontSize="12" />
+          <Panel>
+            <Border Background="{DynamicResource RegionColor}" BoxShadow="{DynamicResource BoxShadow}" IsHitTestVisible="False" />
+            <DockPanel Margin="20 10 20 0">
+                <TextBlock Text="{Binding NavBar.SelectedItem.Title}" DockPanel.Dock="Top" FontSize="28" IsHitTestVisible="False" />
+              <rxui:RoutedViewHost Router="{Binding Router}">
+                <rxui:RoutedViewHost.ViewLocator>
+                  <vl:ScreenViewLocator />
+                </rxui:RoutedViewHost.ViewLocator>
+                <rxui:RoutedViewHost.PageTransition>
+                  <CrossFade Duration="0:0:0.25" />
+                </rxui:RoutedViewHost.PageTransition>
+                <rxui:RoutedViewHost.DefaultContent>
+                  <TextBlock Text="Default content"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center" />
+                </rxui:RoutedViewHost.DefaultContent>
+              </rxui:RoutedViewHost>
+            </DockPanel>
+          </Panel>
+        </DockPanel>
+      </Panel>
+    </SplitView>
+    <c:Dialog Content="{Binding CurrentDialog}" IsDialogOpen="{Binding CurrentDialog.IsDialogOpen, Mode=TwoWay}"/>
```

Comment:
Try and get used to Binding datacontext on toplevel items.

`<c:Dialog DataContext="{Binding CurrentDialog}" IsDialogOpen="{Binding IsDialogOpen}" />`

You shouldnt even need to set content then!

Remember... if you see a '.' in a binding expression, something can be made better.

Reply:
this doesnt work for some reason...

Reply:
Im willing to bet it does ;)

Reply:
@danwalmsley only worked when i added `Content="{Binding }` :)